astropy/wcs/tests/test_wcs.py

```diff
@@ -1075,10 +1068,10 @@ def test_tpv_ctype_tpv():
 
 def test_tpv_ctype_tan():
     sip_header = fits.Header.fromstring(
-        get_pkg_data_contents(os.path.join("data", "siponly.hdr"), encoding="binary")
+        get_pkg_data_contents(Path("data") / "siponly.hdr", encoding="binary")
```

Comment:
maybe consolidate the `Path("data")` here and below into a variable?
Actually, make it a pytest fixture! It's used all over this file.

Reply:
Actually! put it in `wcs/tests/conftest.py` then it can be used across all the `wcs/tests`.

Reply:
Hum, I'm not convinced that such a small fixture is worth adding an indirection to all tests that would use it.

Reply:
Sure.